src/llm/http_llm_calculator.cc

```diff
@@ -516,10 +520,9 @@ class HttpLLMCalculator : public CalculatorBase {
                 } else {
                     // Beam search only supported for unary
                     std::vector<std::string> completions;
-                    for (GenerationOutput& out : generationOutput) {
+                    for (ov::genai::GenerationOutput& out : generationOutput) {
                         std::vector<int64_t> tokens = out.generated_token_ids;
-                        std::shared_ptr<Tokenizer> tokenizer = nodeResources->cbPipe->get_tokenizer();
-
+                        std::shared_ptr<ov::genai::Tokenizer> tokenizer = std::make_shared<ov::genai::Tokenizer>(nodeResources->cbPipe->get_tokenizer());
```

Comment:
Is it necessary? Seems like this new variable is only used in this block. 
I think in the block below we can use:
`nodeResources->cbPipe->get_tokenizer().decode(tokens)` 
and remove this line completely

Reply:
The way Milosz suggests would decrease number from copies of tokenizer from 2 to 1